modules/tofi/hm.nix

```diff
@@ -34,8 +34,8 @@
         default-result-background = background;
         selection-color = selection;
         selection-background = background;
-        border-width = lib.mkDefault 4;
-        outline-width = lib.mkDefault 2;
+        border-width = 4;
+        outline-width = 2;
```

Comment:
These were added in https://github.com/danth/stylix/pull/410, I think it makes sense for them to be a special case since they aren't options Stylix would normally set